source/production/F0.Minesweeper.Server/Pages/Index.razor

```diff
@@ -4,5 +4,6 @@
 
 Welcome to Minesweeper, powered by Blazor.
 
+<Difficulty OnChange="OnDifficultyChange" />
```

Comment:
Seeing this added here in the `Index.razor` in combination with the `Minefield` component makes me think that we need some kind of `Game` component were we put those kinds of things for the game together

Reply:
```suggestion
<Difficulty OnChange="OnDifficultyChange" />
```

:) 
At this point, I wasn't even sure where the difficulty selector should be.
Is it part of the navigation?
Is it part of the header? - sub- navigation kind of thing
Or as you say, part of a Game component. 

Do we want a Game component that centralizes the UI logic. 
Kind of a manager that controls all other components?

I'm also curious if there are other means of communication between components
Like a publisher/subscriber, bus kind of thing